lib/spack/spack/directory_layout.py

```diff
@@ -88,34 +89,38 @@ def check_installed(self, spec):
         """
         raise NotImplementedError()
 
-    def extension_map(self, spec):
-        """Get a dict of currently installed extension packages for a spec.
+    def extension_map(self, spec, path_view=""):
```

Comment:
>However, if we decide against globally activated extensions (which I would prefer!) this would be a non-issue and I'd get started right away.

To be clear, when you say that do you mean "not mixing global and view extensions" rather than "disabling support for global extensions altogether"? If so then I agree, and I think that is consistent with the conversation at https://github.com/LLNL/spack/pull/3227/files#r113287441